src/content/docs/adventure/platform/modded.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Modded (Fabric and NeoForge shared API)
+description: Fabric and NeoForge shared Adventure implementation.
+---
+
+import { Tabs, TabItem } from "@astrojs/starlight/components";
+
+Starting with *Minecraft: Java Edition* 1.21.2, Adventure is implemented using mostly shared code between NeoForge and Fabric.
+Each major version of Minecraft will usually require a new release of the platform.
+
+The platform supports all features, including localization and custom renderers.
+
+## Dependency
+
+When building multi-loader mods, we often want to move as much code as possible into a loader-agnostic part of our projects.
+
+Adventure facilities this through the `net.kyori:adventure-platform-mod-shared` artifact.
+
+A second variant, `net.kyori:adventure-platform-mod-shared-fabric-repack`, is also published. This variant should be used
+when your common code is managed by `fabric-loom`.
+
+If you are building a Fabric-only or NeoForge-only mod, or do not care to use the platform API in shared code,
+then you don't need to use this artifact explicitly. This is because both platforms depend on it transitively.
+
+As with the rest of the Adventure projects, releases are distributed on Maven Central, and snapshots on Sonatype OSS:
+
+<Tabs syncKey="build-system">
+  <TabItem label="Gradle (Kotlin)">
+    ```kotlin title="build.gradle.kts" replace
+    repositories {
+      // for development builds
+      maven(url = "https://s01.oss.sonatype.org/content/repositories/snapshots/") {
+        name = "sonatype-oss-snapshots1"
+        mavenContent { snapshotsOnly() }
+      }
+      // for releases
+      mavenCentral()
+    }
+
+    dependencies {
+      // Loom project
+      modCompileOnly("net.kyori:adventure-platform-mod-shared-fabric-repack:\{LATEST_ADVENTURE_PLATFORM_MOD_RELEASE}") // for Minecraft 1.21.5
```

Comment:
Why specifically 1.21.5 if we are using a placeholder

Reply:
in the current documentation site, those versions are manually edited by the Adventure team on release and the placeholder version is also bumped manually (our port automatically gets the latest one though, maybe revert that to the old behavior?)

Reply:
the original placeholder versioning behavior has been restored and the MC version has been moved to a placeholder as well